### DISCUSSION TITLE
[Have to remove]

### DISCUSSION THREAD

**Discussion by AugustinMauroy** [mdn/mdn-community#136] [Content]
# Tutoriel complet sur NodeJS
Bonjour à tous, je crée cette discussion pour savoir si des personnes seraient en mesure de maintenir le tutoriel sur Node.JS.

Le tutoriel permettrais d'apprendre l'utilisation de Nodejs.

Etant francophone, je peux faire la traduction de ces documents.

# Complete NodeJS tutorial
Hello everyone, I create this discussion to know if people would be able to maintain the tutorial on Node.JS.

The tutorial would allow to learn how to use Nodejs.

Being French speaking, I can make the translation of these documents.

**Comment by cw118** [comment]
Bonjour, la traduction/localisation de la documentation MDN se passe sur [`mdn/translated-content`](https://github.com/mdn/translated-content) et il y a une équipe de revue pour chaque locale. Vous pouvez apporter des pull request sur le dépôt `translated-content` pour traduire/update des pages françaises. Merci déjà pour tout le travail :)

**Reply by SphinxKnight** [reply]
Bienvenue @AugustinMauroy, comme indiqué par @cw118, n'hésitez pas à faire des PR sur https://github.com/mdn/translated-content, voire à ouvrir ces PR en brouillon si vous avez besoin d'un coup de main :) 
Au cas où, voici un petit guide sur la contribution https://tech.mozfr.org/post/2021/03/16/MDN-sur-GitHub-comment-contribuer

**Reply by AugustinMauroy** [reply]
It had been closed by someone because it was necessary to pass through here to then open the pr.

**Comment by schalkneethling** [comment]
Notifying @mdn/core-yari-content so they are aware of this discussion.

**Reply by AugustinMauroy** [reply]
What is @mdn/core-yari-content ?

**Reply by schalkneethling** [reply]
It is the main team members responsible for writing and maintaining the content of MDN Web Docs.

**Comment by GempitaRizki** [comment]
Thank you so much for sharing; it's incredibly useful.

**Comment by schalkneethling** [comment]
Just so that everyone has some more context around this topic, please also read [this comment](https://github.com/mdn/content/pull/17627#issuecomment-1171112696) as well as [this comment](https://github.com/mdn/content/pull/17627#issuecomment-1166590481) by @wbamberg - Thank you!
